### ISSUE TITLE
MACROLET accepts anything as a name

### ISSUE THREAD

**Issue by stassats** [drmeister/clasp#208]
(macrolet ((123 ()))) is accepted.

**Comment by Shinmera** [comment]
Cannot confirm on b8c6690535bdc61bfdaeb50dbe23cd89633200a5
```
* (macrolet ((123 ())))

debugger invoked on a SB-INT:SIMPLE-PROGRAM-ERROR in thread
#<THREAD "main thread" RUNNING {1002B1C573}>:
  The local macro name 123 is not a symbol.
```

**Comment by Shinmera** [comment]
Fixed on 99669d674a6f2b7ce0a9127cf6537fc03066baa5
```
> (macrolet ((123 ())))

Condition of type: SIMPLE-PROGRAM-ERROR
Invalid function name: 123
```
